cmd/build.go

```diff
@@ -12,11 +12,22 @@ import (
 	"github.com/elastic/elastic-package/internal/docs"
 )
 
+const buildLongDescription = `Use this command to build a package. Currently it supports only the "integration" package type.
+
+Built packages are stored in the "build/" folder located at the root folder of the local Git repository checkout that contains your package folder. The command will also render the README file if there is a corresponding template file present in "_dev/build/docs/README.md". All "_dev" directories will be omitted.
+
+Built packages are served up by the Elastic Package Registry running locally (see "elastic-package stack"). If you want a local package to be served up by the local Elastic Package Registry, make sure to build that package first using "elastic-package build".
+
+Built packages can also be published to the "package-storage" repository.
```

Comment:
Let's not try to reference package-storage as much as possible, as it should be more of an implementation detail IMO, especially as we think about redesigning the package registry in the future.

```suggestion
Built packages can also be published to the global package registry service.
```

Reply:
Good idea, fixed